src/Softeq.NetKit.Chat.Data.Persistent.Sql/Repositories/MemberRepository.cs

```diff
@@ -204,10 +278,25 @@ public async Task<IReadOnlyCollection<Member>> GetAllMembersByChannelIdAsync(Gui
         {
             using (var connection = _sqlConnectionFactory.CreateConnection())
             {
-                var sqlQuery = @"SELECT m.Id, m.Email, m.IsAfk, m.IsBanned, m.LastActivity, m.LastNudged, m.Name, m.PhotoName, m.Role, m.SaasUserId, m.Status, m.IsDeleted  
-                                 FROM Members m
-                                 INNER JOIN ChannelMembers c ON m.Id = c.MemberId
-                                 WHERE c.ChannelId = @channelId";
+                var sqlQuery = $@"
+                    SELECT 
+                        m.{nameof(Member.Id)}, 
+                        m.{nameof(Member.Email)}, 
+                        m.{nameof(Member.IsAfk)}, 
+                        m.{nameof(Member.IsBanned)}, 
+                        m.{nameof(Member.LastActivity)}, 
+                        m.{nameof(Member.LastNudged)}, 
+                        m.{nameof(Member.Name)}, 
+                        m.{nameof(Member.PhotoName)}, 
+                        m.{nameof(Member.Role)}, 
+                        m.{nameof(Member.SaasUserId)}, 
+                        m.{nameof(Member.Status)}
```

Comment:
We need to add ChannelMembers into SELECT statement as we are using INNER JOIN here.

Reply:
No, we didn't